client/src/App.jsx

```diff
@@ -1,10 +1,14 @@
 import { Outlet } from "react-router-dom";
 import Footer from "./components/Footer";
+import Header from "./components/Header";
```

Comment:
you should have a layout folder with a index.tsx file containing header and footer , that layout component should then be in app.tsx, cause header and footer are layout components not generic components that are reused